app/src/main/java/org/fossasia/openevent/general/auth/ProfileViewModel.kt

```diff
@@ -38,6 +45,43 @@ class ProfileViewModel(private val authService: AuthService, private val resourc
             }
     }
 
+    fun checkIfAccountCanBeDeleted() {
+        val query = """[{
+                |   'and':[{
+                |       'name':'is-event-online',
+                |       'op':'eq',
+                |       'val': 'true'
+                |    }]
+                |}]""".trimMargin().replace("'", "\"")
+
+        compositeDisposable += eventService.getEventsOfUser(authService.getId(), query)
+            .withDefaultSchedulers()
```

Comment:
This is not your job. This is server's job

Reply:
I thought frontend also did the same. There is no endpoint to check whether an account is deletable or not, so we have to check whether there are active events attached to the account from client-side or not

Reply:
So should I just proceed to delete and then when server return delete fail, I'll display the message that there are events attached so the account is not deletable?

Reply:
yes